### PR TITLE
[test] SwiftFixIt: Add XFAILed test for duplicate insertion fix-its

### PR DESCRIPTION
**Pull request by AnthonyLatsis**
### Motivation:

We need to deduplicate these fix-its. In the meantime, commit the behavioral expectation.

### Modifications:

New test case

### CODE REVIEW

**Comment by AnthonyLatsis** [comment]
@swift-ci please test

File: Tests/SwiftFixItTests/FilteringTests.swift [hunk 1]
```diff
@@ -411,4 +411,126 @@ struct FilteringTests {
...
+
+    @Test
+    func testDuplicateInsertionFixIts() throws {
+        withKnownIssue("FIXME: Filter out duplicate insertion fix-its") {
+            try self._testDuplicateInsertionFixIts()
+        }
+    }
+
+    func _testDuplicateInsertionFixIts() throws {
+        try testAPI1File { path in
```
Comment (AnthonyLatsis) [inline Tests/SwiftFixItTests/FilteringTests.swift:423]: https://github.com/swiftlang/swift-testing/issues/1132
